docs/modules/high_level.rst

```diff
@@ -1,17 +1,28 @@
-.. _high_level:
-
-
 High Level 
 ==========
 
-.. autoclass:: jaxsim.high_level.model.Model
+.. currentmodule:: jaxsim.high_level
+
+
+Model
+~~~~~
+
+.. automodule:: jaxsim.high_level.model
     :members:
-    :inherited-members:
 
-.. autoclass:: jaxsim.high_level.joint.Joint
+Joint
+~~~~~
+
+.. automodule:: jaxsim.high_level.joint
+    :members:
+
+Link
+~~~~~
+.. automodule:: jaxsim.high_level.link
     :members:
-    :inherited-members:
 
-.. autoclass:: jaxsim.high_level.link.Link
+Common
+~~~~~~
+
+.. automodule:: jaxsim.high_level.common
```

Comment:
@flferretti is there a way to make the enum show up? I can only see the following from the preview:
![Screenshot_20240206_092447](https://github.com/ami-iit/jaxsim/assets/469199/02444bec-abd4-4bfd-a058-1378787fc3ca)

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/79/commits/0d50cb5b7bcb580e906411a753faf569ae0ec317